### PR TITLE
Manage tables

### PR DESCRIPTION
**Pull request by Matheus-OAMK**

### CODE REVIEW

File: src/controllers/table.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    const existingRestaurant = await getRestaurantById(restaurantId);
+
+    if (!existingRestaurant) {
+      return sendResponse(res, 404, "no restaurant found");
+    }
+
+    // TODO CHECK IF USER HAS PERMISSION
+    if (existingRestaurant.business_id !== req.user.id) {
+      return sendResponse(res, 403, "invalid session");
+    }
```
Comment (RepoRover) [inline src/controllers/table.ts:140]: It says TODO, but seems like it is already done, right?

**Review by RepoRover — COMMENTED** [review]
When feature/restaurant-profile-management branch will get merged into main, you will need to merge changes from main branch into feature/manage-tables and manage-tables branches, so this way we ensure those two branches that are present in this pull request have new changes from main as it might cause conflicts when we try to pull feature/manage-tables into main. It is actually should be a rule that if you have a pull requests open, make sure all the branches that are present in the pull are up to date with main

File: src/controllers/table.ts (hunk 1 above)
Comment (Matheus-OAMK) [reply src/controllers/table.ts:140]: Yes, old comment

**Comment by Matheus-OAMK** [comment]
> When feature/restaurant-profile-management branch will get merged into main, you will need to merge changes from main branch into feature/manage-tables and manage-tables branches, so this way we ensure those two branches that are present in this pull request have new changes from main as it might cause conflicts when we try to pull feature/manage-tables into main. It is actually should be a rule that if you have a pull requests open, make sure all the branches that are present in the pull are up to date with main, and also this way will be actually able to see what has been done in this current branch as the changes that you made in another branch are present here, so when we merge that branch with main, and then merge main branch changes into your feature and task branches, I will see all the current task and feature updates only

This branch is currently up to date with both those branches technically

**Comment by RepoRover** [comment]
> > When feature/restaurant-profile-management branch will get merged into main, you will need to merge changes from main branch into feature/manage-tables and manage-tables branches, so this way we ensure those two branches that are present in this pull request have new changes from main as it might cause conflicts when we try to pull feature/manage-tables into main. It is actually should be a rule that if you have a pull requests open, make sure all the branches that are present in the pull are up to date with main, and also this way will be actually able to see what has been done in this current branch as the changes that you made in another branch are present here, so when we merge that branch with main, and then merge main branch changes into your feature and task branches, I will see all the current task and feature updates only
> 
> This branch is currently up to date with both those branches technically

Now yes, that's why I say "When feature/restaurant-profile-management branch will get merged into main" and not now

**Comment by Matheus-OAMK** [comment]
> > > When feature/restaurant-profile-management branch will get merged into main, you will need to merge changes from main branch into feature/manage-tables and manage-tables branches, so this way we ensure those two branches that are present in this pull request have new changes from main as it might cause conflicts when we try to pull feature/manage-tables into main. It is actually should be a rule that if you have a pull requests open, make sure all the branches that are present in the pull are up to date with main, and also this way will be actually able to see what has been done in this current branch as the changes that you made in another branch are present here, so when we merge that branch with main, and then merge main branch changes into your feature and task branches, I will see all the current task and feature updates only
> > 
> > 
> > This branch is currently up to date with both those branches technically
> 
> Now yes, that's why I say "When feature/restaurant-profile-management branch will get merged into main" and not now

Well if u notice this branch had all commits from the feature/restaurant-profile-management branch ... 
This is no longer true because of yesterday's changes.

**Comment by RepoRover** [comment]
> > > > When feature/restaurant-profile-management branch will get merged into main, you will need to merge changes from main branch into feature/manage-tables and manage-tables branches, so this way we ensure those two branches that are present in this pull request have new changes from main as it might cause conflicts when we try to pull feature/manage-tables into main. It is actually should be a rule that if you have a pull requests open, make sure all the branches that are present in the pull are up to date with main, and also this way will be actually able to see what has been done in this current branch as the changes that you made in another branch are present here, so when we merge that branch with main, and then merge main branch changes into your feature and task branches, I will see all the current task and feature updates only
> > > 
> > > 
> > > This branch is currently up to date with both those branches technically
> > 
> > 
> > Now yes, that's why I say "When feature/restaurant-profile-management branch will get merged into main" and not now
> 
> Well if u notice this branch had all commits from the feature/restaurant-profile-management branch ... This is no longer true because of yesterday's changes.

Yes, you're right, the thing is that I don't see the exact changes related to this task, I can see 1100 lines added when it is clearly not true for this task, I can find changes for this task only with careful checking, but it is very inconvenient, so, when you will merge new changes from main into two branches that are here, in this pull, I will see only changes related to this task

**Review by RepoRover — APPROVED** [review]
Well done!

**Pull request merged**